cookie.go

```diff
@@ -18,3 +28,66 @@ func (c *Client) GetWebCookieRefreshInfo() (*GetWebCookieRefreshInfoResult, erro
 
 	return execute[*GetWebCookieRefreshInfoResult](c, method, url, nil)
```

Comment:
这里需要填入csrf吗？我看文档中是非必填。如果你想要填上就改成：
```suggestion
	return execute[*GetWebCookieRefreshInfoResult](c, method, url, fillCsrf)
```